design/components/card.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Story } from '@storybook/react/types-6-0';
+import React from 'react';
+
+import duurzaamheid from './assets/duurzaamheid.png';
+import { CardProps, Card } from './card';
+import { RichText } from './rich-text';
+import { Tag } from './tag';
+import { Title } from './title';
+
+export default {
+  title: 'Example/Card',
+  component: Card,
+};
+
+const Template: Story<CardProps> = (args): React.ReactElement => (
+  <Card {...args} />
+);
```

Comment:
FYI you can wrap the Card in the story template eg. like this.

```ts
const Template: Story<CardProps> = (args): React.ReactElement => (
  <div style={{ maxWidth: 500 }}>
    <Card {...args} />
  </div>
);
```

If you want to limit the width of the card without doing it in the component.